### ISSUE TITLE
dataset documentation 2 - alias data types

### ISSUE THREAD

**Issue by lwaldron** [waldronlab/curatedMetagenomicData#17]
Documenting 6 different datasets for each study is overly verbose, it would be better for them to share one .Rd page and use \alias{} for that page to be shared by all six data types for the study. For example, see this old [Candela_Africa.stool.eset.Rd](https://github.com/vobencha/curatedMetagenomicData/blob/397fe7b75397d2170b2759f4b1509e9be3bac59a/man/Candela_Africa.stool.eset.Rd).

**Comment by schifferl** [comment]
agreed, I will rework the documentation to reflect the slimed down and more informative style

**Comment by lwaldron** [comment]
I'll add this note just to make sure that the row counts are corrected in the update (I think these counts are from before removing per-species contributions to gene families):

```
HMP_2012.genefamilies_relab.anterior_nares: An ExpressionSet with 93 samples and 1,964,481
features specific to the anterior_nares bodysite
```

**Comment by schifferl** [comment]
Great! I will check and make sure the counts are correct as I rebuild the documentation

**Comment by lwaldron** [comment]
@schifferl the new Bioconductor release is around the corner - would you put outstanding issues on your Trello board with due dates to get them done before the release? I'm just preparing to present the package tomorrow and realized that the datasets still aren't aliased and don't contain the enriched documentation that my original documentation generator provided.  We're likely also to have work from reviewers by then, so don't leave it too close :).

**Comment by schifferl** [comment]
@lwaldron it was a bit of a busy week last week but I will get the issues on Trello this morning and make sure the important one's are done for the Bioconductor release. Sorry for the delay.

**Comment by schifferl** [comment]
Closed by f535e99e53099ed03b0914f1e36c59b7cdb20ebb, simply github install the package to see the more informative and condensed documentation
